aws/resource_aws_apigatewayv2_api.go

```diff
@@ -135,6 +141,39 @@ func resourceAwsApiGatewayV2Api() *schema.Resource {
 	}
 }
 
+func resourceAwsAPIGatewayV2ImportOpenAPI(d *schema.ResourceData, meta interface{}) error {
+	conn := meta.(*AWSClient).apigatewayv2conn
+
+	if body, ok := d.GetOk("body"); ok {
+		revertReq := &apigatewayv2.UpdateApiInput{
```

Comment:
Can you please note in the documentation that `name`, `description` and `version` should be specified in the Terraform configuration and the values will override any values specified in the OpenAPI document.
`cors_configuration` - https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-swagger-extensions-cors-configuration.html - and `tags` - https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-openapi-extensions-x-amazon-apigateway-tag-value.html - can also be specified in the OpenAPI document so we should have the same behaviour for those attributes.